### PR TITLE
Improve README.md readability

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Sequestrum
 ## Description
+<<<<<<< HEAD
```
Comment (buffet) [inline README.md]: This shouldn't be in here.

**Review by buffet — CHANGES_REQUESTED** [review]

**Review by buffet — APPROVED** [review]

**Review by trvv — APPROVED** [review]

**Comment by iiPlasma** [comment]
As you friend @goteemLight , I'm going to have to close this for the memes. #HacktoberfestDenied

**Comment by buffet** [comment]
@iiPlasma Hacktoberfest doesn't care if something gets closed, as long as it's not a troll.

**Comment by iiPlasma** [comment]
@buffet This is pretty troll, gonna report real quick

**Pull request closed without merging**